main.tf

```
@@ -64,6 +73,22 @@ locals {
       name      = "${local.secret_webhook_key}"
       valueFrom = "${var.webhook_ssm_parameter_name}"
     },
+    {
+      name      = "${local.akamai_client_secret_key}"
+      valueFrom = "${local.akamai_client_secret_value_from}"
```

I'm not sure why the `valueFrom` need to be locals. Couldn't they just pull from the variables directly?

```suggestion
      valueFrom = "${var.akamai_client_token_parameter_name}"
```